x/supplier/client/cli/query_claim.go

```diff
@@ -0,0 +1,78 @@
+package cli
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/spf13/cobra"
+
+	"github.com/pokt-network/poktroll/x/supplier/types"
+)
+
+func CmdListClaim() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list-claim",
+		Short: "list all claim",
```

Comment:
Wdyt about explicitly indicating that no arguments  are accepted?:

```suggestion
		Short: "list all claim",
		Args: cobra.NoArgs(),
```